code/game/objects/items/weapons/vending_items.dm

```diff
@@ -71,3 +71,9 @@
 	icon_state = "refill_clothes"
 	charges = list(35, 13, 4)// of 101 standard, 12 contraband, 10 premium(?)
 	init_charges = list(35, 13, 4)
+
+/obj/item/weapon/vending_refill/kink
+	machine_name 	= "KinkMate"
+	icon_state 		= "refill_kink"
+	charges 		= list(15, 15, 4)// of 30 standard, 0 contraband, 15 premium
```

Comment:
These charges need to be just above 1/3 of the total in the machine, so it should be
`charges 		= list(6, 3, 0)// of 15 standard, 6 contraband, 0 premium`
Likewise with `init_charges`. Just follow the pattern of the other refillers.